hw_3/script.js

```diff
@@ -0,0 +1,111 @@
+function checkArgLength(args) {
+    if (args.length !== 2) {
+        throw new Error("Not 2 arguments");
+    }
+}
+
+function checkMinMax(min, max) {
+    if (!Number.isInteger(min) || !Number.isInteger(max)) {
+        throw new Error("arguments are not Integer");
+    }
+    if (min > max) {
+        throw new Error("Min > Max");
+    }
+}
+
+function checkResult(result) {
```

Comment:
Perhaps this function name would make more sense and the body of the function itself could be shortened slightly.
```suggestion
function checkValueForMaxSafeInteger(result) {
    if (result > Number.MAX_SAFE_INTEGER) {
        return new Error(`The value is greater than ${Number.MAX_SAFE_INTEGER}`)
    }

    return result;
}
```

Reply:
Renamed function.
https://github.com/BlackTruth/javascript/blob/813db56a38899f41dcfdc4b332943bde5c271592/hw_3/script.js#L1